package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@metamask/controllers-monorepo",
-  "version": "33.0.0",
+  "version": "34.0.0",
```

Comment:
Should this be `-rc.1` as well?

Reply:
Hmm... Yes, I think that makes sense given this is a special release. Updated in [0e5cd32](https://github.com/MetaMask/controllers/pull/960/commits/0e5cd329f2ff7c07a7d0c3cdbc7549f3a56a3153).